### PR TITLE
Makes nerve splicing surgery reduce stamina damage

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Makes nerve splicing surgery also reduce stamina damage by 20% (amount of stun/stamina reduction can be changed if requested).

## Why It's Good For The Game

Most of stun sources now come from stamina damage, so this surgery is kinda useless in most cases. This PR makes it more usefull in the current combat system, while not making it too OP.

## Changelog

:cl:
balance: Nerve splicing surgery now additionaly reduces stamina damage taken by 20%
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
This reduces the baton's damage from 60 to 45, making it three hits. This is actually rather massive considering the current (basically head-coder enforced) meta.

I guess for data, the disabler goes from 30 to 22.5, so an extra hit. Doesn't really matter too much since without armor it still only takes two hits for it to slow you down.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Absolutely 0 opinion on the balance change, but just a warning

**Comment by SuperSlayer0** [comment]
> This reduces the baton's damage from 60 to 45, making it three hits. This is actually rather massive considering the current (basically head-coder enforced) meta.
> 
> I guess for data, the disabler goes from 30 to 22.5, so an extra hit. Doesn't really matter too much since without armor it still only takes two hits for it to slow you down.

It seems OK since it is a lategame surgery, and requires(impossible) social interactions with someone to get it, so it still wouldn't be seen often.

File: code/modules/surgery/advanced/bioware/nerve_splicing.dm [hunk 1]
```diff
@@ -49,7 +49,9 @@
 /datum/bioware/spliced_nerves/on_gain()
 	..()
 	owner.physiology.stun_mod *= 0.5
+	owner.physiology.stamina_mod -= 0.25
```
[the remark this answers is not in the document]
Comment (SuperSlayer0) [reply code/modules/surgery/advanced/bioware/nerve_splicing.dm]: Done

**Comment by improvedname** [comment]
it can be researched roundstart tho lol

**Comment by Serxule** [comment]
could this be done with CNS rebooter too? afaik it does absolutely nothing unless you get shoved down

**Comment by VladinXXV** [comment]
> could this be done with CNS rebooter too? afaik it does absolutely nothing unless you get shoved down

it definitely has a big effect on goliath tentacle stuns but thats all i can say for sure

**Comment by VladinXXV** [comment]
> This reduces the baton's damage from 60 to 45, making it three hits. This is actually rather massive considering the current (basically head-coder enforced) meta.
> 
> I guess for data, the disabler goes from 30 to 22.5, so an extra hit. Doesn't really matter too much since without armor it still only takes two hits for it to slow you down.

image
image
the intended effect is reducing stun time by 50%, i think one extra hit in the form of 20% less stamina damage is quite mild compared to the more obvious 50% reduction that could be imposed

**Comment by Serxule** [comment]
> image image the intended effect is reducing stun time by 50%, i think one extra hit in the form of 20% less stamina damage is quite mild compared to the more obvious 50% reduction that could be imposed

could just be changed to not say "greatly reduced", because honestly a baton taking 4 hits to stun someone compared to 2 is a very big (definitely unbalanced) difference especially with how the server currently is

**Comment by VladinXXV** [comment]
> > image image the intended effect is reducing stun time by 50%, i think one extra hit in the form of 20% less stamina damage is quite mild compared to the more obvious 50% reduction that could be imposed
> 
> could just be changed, because honestly a baton taking 4 hits to stun someone compared to 2 is a very big (definitely unbalanced) difference especially with how the server currently is

my point isn't "please make it 4 hits" my point is "1 additional hit isn't the end of the world when we could just double everything"

**Review by Timberpoes — CHANGES_REQUESTED** [review]
There are two ways to find out the ramifications. Fullmerge and force the players to deal with it, and testmerge to playtest if people notice and start trying to rush the surgery.

Surgery requires trusting someone else. Things that require trusting someone else can always be a little crazy, because that pushes character interactions. Especially when you make yourself a little vulnerable in the process of trusting someone else.

Can you add blackbox feedback recording to when people successfully receive this surgery, recording every ckey that gets the surgery?

This record type should be suitable:
image

It would enable a bit of data analysis if it interested anyone, because I definitely think this is 100% capable of being merged. But I'll probably look to testmerge it first and see if it ends up fucking shit up or something.

**Comment by moocowswag** [comment]
How does this interact with the extra stamina damage you take when you enter stam crit?

I think the combination of this surgery + stamina regen chems might make stamina regeneration viable to an extent again.

**Comment by SuperSlayer0** [comment]
> Can you add blackbox feedback recording to when people successfully receive this surgery, recording every ckey that gets the surgery?
> 
> This record type should be suitable: image
> 
> It would enable a bit of data analysis if it interested anyone, because I definitely think this is 100% capable of being merged. But I'll probably look to testmerge it first and see if it ends up fucking shit up or something.

Okay, i will do this

**Comment by Overuseddart** [comment]
> 

You can still shove then baton, the effect on batons is minimal (except for telebatons)

**Review by Timberpoes — APPROVED** [review]
Across 200 testmerged shifts on Sybil and Manuel, the surgery itself has only been gotten in 18 of them.

Of those 18, most most only had 1 player get the surgery. The max was 4 players.

I think it's safe to say that this doesn't look like it shall radically change the meta. Surgeries which require cooperation and making yourself potentially vulnerable are good sources of extra power in a shift.

**Pull request merged**